src/scss/survey.scss

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import 'includes/mixins';
+@import 'includes/variables';
+
+.survey-banner {
+    @include gradient-horizontal($start-color: #ff00ff, $end-color: #0001fe);
```

Comment:
Do we need a solid background color as a fallback? Though gradients are really well supported, maybe not an issue for our audience.